ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.collection.view.js

```diff
@@ -28,7 +25,6 @@ module.exports = Marionette.CollectionView.extend({
   childViewOptions: function() {
     return {
       queryModel: this.options.queryModel,
-      sharingLightboxTitle: 'Search Form Sharing',
```

Comment:
❓ Why are we getting rid of this?

Reply:
IIRC it didn't work and that was the only value ever used for that option so I just inlined it